pallets/unified-accounts/src/impls.rs

```diff
@@ -0,0 +1,175 @@
+// This file is part of Astar.
```

Comment:
What's the benefit of having a separate `impl.rs` file? It's not modular from `Pallet` anyway. One has to jump back and forth between this and `lib.rs` to understand what's happening.

Reply:
To keep the pallet code clean and not clutter it with different trait implementations some of them are not entirely related to the pallet itself

Reply:
I'll move the implementations that are dependent  on pallet itself to `lib.rs`